### PR TITLE
Rectangles exercise

### PR DESCRIPTION
**Pull request by glennj**
I'll stick to the "desirables" list.

### CODE REVIEW

**Comment by joshgoebel** [comment]
Desirables probably means "my personal favs" so don't let that hold ya back too much. 🔌

File: exercises/practice/rectangles/package.wren [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import "wren-package" for WrenPackage, Dependency
+import "os" for Process
+
+class Package is WrenPackage {
+  construct new() {}
+  name { "exercism/hamming" }
```
[the remark this answers is not in the document]
Comment (glennj) [reply exercises/practice/rectangles/package.wren:6]: Do these need changing too?
```
$ ag exercism-wren-track exercises/practice/*/package.wren 
exercises/practice/clock/package.wren
6:  name { "exercism-wren-track" }

exercises/practice/complex-numbers/package.wren
6:  name { "exercism-wren-track" }

exercises/practice/grep/package.wren
6:  name { "exercism-wren-track" }

exercises/practice/isogram/package.wren
6:  name { "exercism-wren-track" }

exercises/practice/rational-numbers/package.wren
6:  name { "exercism-wren-track" }
```

File: exercises/practice/rectangles/.meta/proof.ci.wren [hunk 2]
```diff
@@ -0,0 +1,66 @@
+class Rectangles {
+  construct new(lines) {
+    _board = lines.map {|line| line.bytes}.toList
```
[the remark this answers is not in the document]
Comment (glennj) [reply exercises/practice/rectangles/.meta/proof.ci.wren:3]: I sometimes forget a String is a Sequence.

File: exercises/practice/rectangles/package.wren (hunk 1 above)
[the remark this answers is not in the document]
Comment (joshgoebel) [reply exercises/practice/rectangles/package.wren]: Yes.

File: exercises/practice/rectangles/.meta/proof.ci.wren (hunk 2 above)
[the remark this answers is not in the document]
Comment (joshgoebel) [reply exercises/practice/rectangles/.meta/proof.ci.wren]: Are you trying to copy with the indexing?  Aren't string immutable?

[the remark this answers is not in the document]
Comment (glennj) [reply exercises/practice/rectangles/.meta/proof.ci.wren]: I rebased pig-latin on this branch, which made it messy. But are you referring to the current constructor which does

    _board = lines[0..-1]

Yes the strings are immutable, but the array is not. I'm copying the contents of the `lines` array.

[the remark this answers is not in the document]
Comment (joshgoebel) [reply exercises/practice/rectangles/.meta/proof.ci.wren]: Ah right, lines duh... can we rebase it without the piglatin?

**Comment by glennj** [comment]
OK, clean PR now

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
@glennj Thanks again!

**Pull request merged**